pkg/oc/clusteradd/componentinstall/apply_template.go

```diff
@@ -130,7 +130,7 @@ func (opt installReadyTemplate) Install(dockerClient dockerhelper.Interface) err
 		return nil
 	}
 
-	if err := wait.PollImmediate(time.Second, 5*time.Minute, opt.template.WaitCondition); err != nil {
+	if err := wait.PollImmediate(time.Second, 20*time.Minute, opt.template.WaitCondition); err != nil {
```

Comment:
Yeah, you can make this an option.  Unset is still 5 minutes and you can set it really long for logging.